src/Renderer/RendererOpenGL.cpp

```diff
@@ -571,7 +571,7 @@ float RendererOpenGL::height()
 void RendererOpenGL::size(int w, int h)
 {
 	SDL_SetWindowSize(underlyingWindow, w, h);
-	resizeViewport(w, h);
+	setViewport(w, h);
```

Comment:
This commit is focused on adding a new method, which makes this change look out of place.